.vscode/cspell.json

```diff
@@ -110,16 +110,6 @@
     "tools/azure-sdk-tools/tests/test_servicemetadata.py"
   ],
   "words": [
-    "pyprojecttoml",
```

Comment:
Why are these terms being removed? https://github.com/Azure/azure-sdk-for-python/blob/main/.vscode/cspell.json

Reply:
I'm not sure what removed them. Should I manually add them back in?

Reply:
If you did not deliberately remove these items, I would ensure that you are synced with the latest in the official repo.  I would assume no changes to this file would be required.

Reply:
Just synced and merged. No conflicts. I don't see any changes to cspell either ....